src/pages/Checkout/CheckoutPage.vue

```diff
@@ -147,7 +151,7 @@
 					@updating-totals="setUpdatingTotals"
 					@refreshtotals="refreshTotals"
 				/>
-				<loading-overlay v-show="updatingTotals" id="updating-overlay" class="updating-totals-overlay" />
+				<kv-loading-overlay v-if="updatingTotals" id="updating-overlay" class="updating-totals-overlay" />
```

Comment:
There was a small bug here with the way the mounted hook was being fired on v-show which led to the loader sometimes not being in the correct place